scrapi/harvesters/stepic.py

```diff
@@ -0,0 +1,93 @@
+"""
+Stepic.org harvester of MOOC-online courses for SHARE Notification Service
+Example API query: https://stepic.org:443/api/lessons/100
+"""
+
+
+from __future__ import unicode_literals
+
+import json
+import logging
+from dateutil.parser import parse
+import pycountry
+
+from scrapi import requests
+from scrapi.base import JSONHarvester
+from scrapi.linter.document import RawDocument
+
+logger = logging.getLogger(__name__)
+
+
+def process_owner(owners_id):
+    logger.info(owners_id)
+    resp = requests.get("https://stepic.org:443/api/users/" + str(owners_id)).json()
+    try:
+        person = resp[u'users'][0]
+    except KeyError:
+        person = {u'first_name': 'None', u'last_name': 'None'}
```

Comment:
Could you make these empty strings (```''```) instead? Otherwise, we will get contributors whose name displays as "None None", rather than "".